lib/src/content/components/main_content/wolt_modal_sheet_main_content.dart

```diff
@@ -13,15 +13,104 @@ class WoltModalSheetMainContent extends StatelessWidget {
   final SliverWoltModalSheetPage page;
   final WoltModalType woltModalType;
   final WoltModalSheetScrollAnimationStyle scrollAnimationStyle;
+  final WoltModalSheetRoute route;
+  final VoidCallback? onModalDismissedWithDrag;
+  final GlobalKey modalContentKey;
 
   const WoltModalSheetMainContent({
     required this.scrollController,
     required this.pageTitleKey,
     required this.page,
     required this.woltModalType,
     required this.scrollAnimationStyle,
+    required this.route,
+    this.onModalDismissedWithDrag,
+    required this.modalContentKey,
     Key? key,
   }) : super(key: key);
+  RenderBox get _renderBox =>
+      modalContentKey.currentContext!.findRenderObject()! as RenderBox;
+
+  double get _childHeight => _renderBox.size.height;
+  bool get _shouldDragContent =>
```

Comment:
I don't think we should restrict this to `WoltBottomSheetType`. It would be enough to have content draggable.